WooCommerce/Classes/ViewRelated/Reviews/ReviewsViewController.swift

```diff
@@ -291,7 +294,8 @@ private extension ReviewsViewController {
 
         tableView.setContentOffset(.zero, animated: false)
         tableView.reloadData()
-        transitionToSyncingState()
+
+        transitionToSyncingState(pageNumber: SyncingCoordinator.Defaults.pageFirstIndex)
```

Comment:
is this call necessary? since `reloadResultsController` is only called in `defaultSiteWasUpdated` and is followed by `syncingCoordinator.resynchronize()` which triggers syncing the first page, I think it's safe to let the sync callback transition to the expected state after switching sites?

I understand that this call is pre-existing, just thought it might simplify the code a bit by just letting the syncing callback triggers UI state transitions :)

Reply:
It make sense, fixed here 1e26382ce06a7391d4cd4f088d4913837f9e5dd4 🙇